internal/build/src/rollup.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  config,
+  commonJs,
+  typescript,
+  minify,
+  output,
+  input,
+  external,
+  RollupConfig,
+} from '@vertexvis/build-tools';
+
+type Config = {
+  /**
+   * Indicates if the generated config will output both Node and browser
+   * bundles. This is useful for library projects that need to support both
+   * environments.
+   */
+  isMultiPlatform?: boolean;
+};
+
+export function rollupConfig({ isMultiPlatform = false }: Config = {}):
```

Comment:
Not sure if there'd be any easy way to test this so that this doesn't regress?

Reply:
This is a bit difficult to test, but added some rudimentary ones.